examples/components/HELLOWORLD/traininsilo/run.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import glob
 import os
+from distutils.util import strtobool
```

Comment:
The helloworld example doesn't include any meaningful model training, and it is also irrelevant for dp training (we can see we don't even import opacus in this run script). In this case, would that be slightly better to just drop all the changes about dp in this hello world example? Are there any other potential issues it would cause if we do that?

Reply:
Agree! that doesn't make sense to implement DP in the HELLOWORLD example but the `submit.py` is shared by both MNIST and HELLOWORLD example so we have to do the respective changes in the HELLOWORLD example as well else have to put extra `if else` conditions in the submit.py

Reply:
got it, sounds good to me.